test/e2e/validate/validate_test.go

```diff
@@ -153,6 +170,16 @@ func TestValidate(t *testing.T) {
 		})
 		Expect(err).NotTo(HaveOccurred())
 
+		By("Updating custom kyverno config...")
+		err = e2e.GetWithRetry(1*time.Second, 15, func() error {
+			_, err := e2eClient.UpdateClusteredResourceYaml(configMapGVR, CustomKyvernoConfig)
+			if err != nil {
+				return nil
+			}
+			return fmt.Errorf("failed to update configmap: %v", err)
+		})
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
Updating the config with the new image extractors after the controller has started to showcase the dynamic config loading.